core/build.gradle.kts

```diff
@@ -14,34 +14,34 @@ mavenPublicationsPom {
 }
 
 kotlin {
-    infra {
-        // According to https://kotlinlang.org/docs/native-target-support.html
+    applyDefaultHierarchyTemplate {
 
+        // According to https://kotlinlang.org/docs/native-target-support.html
         // Tier 1
-        target("linuxX64")
-        target("macosX64")
-        target("macosArm64")
-        target("iosSimulatorArm64")
-        target("iosX64")
+        this@kotlin.linuxX64()
```

Comment:
Why explicit `this`?

Reply:
If no `this` fires `KotlinNativeTargetsWithHostTests can't be called in this context by implicit receiver. Use the explicit one if necessary`

Reply:
Can you declare targets outside of `applyDefaultHierarchyTemplate` and call the latter without parameters, as it is suggested in docs? https://kotlinlang.org/docs/multiplatform-hierarchy.html#default-hierarchy-template

Reply:
Fixed